auto_ptr warnings

```
g++ -std=c++11 -I/usr/share/R/include    -I../inst/include/ -I.  -I"/usr/local/lib/R/site-library/Rcpp/include" -I"/usr/local/lib/R/site-library/BH/include"   -fpic  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -D_FORTIFY_SOURCE=2 -g -c blpapi_utils.cpp -o blpapi_utils.o
In file included from /usr/local/lib/R/site-library/BH/include/boost/smart_ptr/shared_ptr.hpp:28:0,
                 from /usr/local/lib/R/site-library/BH/include/boost/shared_ptr.hpp:17,
                 from /usr/local/lib/R/site-library/BH/include/boost/date_time/gregorian/greg_month.hpp:14,
                 from /usr/local/lib/R/site-library/BH/include/boost/date_time/gregorian/greg_ymd.hpp:16,
                 from /usr/local/lib/R/site-library/BH/include/boost/date_time/gregorian/greg_calendar.hpp:16,
                 from /usr/local/lib/R/site-library/BH/include/boost/date_time/gregorian/gregorian_types.hpp:19,
                 from blpapi_utils.cpp:23:
/usr/local/lib/R/site-library/BH/include/boost/smart_ptr/detail/shared_count.hpp:395:33: warning: ‘template<class> class std::auto_ptr’ is deprecated [-Wdeprecated-declarations]
     explicit shared_count( std::auto_ptr<Y> & r ): pi_( new sp_counted_impl_p<Y>( r.get() ) )
 
```
And so on...

--- a/src/blpapi_utils.cpp
+++ b/src/blpapi_utils.cpp
@@ -20,6 +20,7 @@
 #include <sstream>
 //#include <iostream>
 #include <algorithm>
+#define BOOST_NO_AUTO_PTR
 #include <boost/date_time/gregorian/gregorian_types.hpp>
 #include <boost/date_time/posix_time/posix_time_types.hpp>
 #include <boost/date_time/local_time/local_time_types.hpp>

